CHANGELOG.md

```diff
@@ -1,5 +1,65 @@
 # Git LFS Changelog
 
+## 1.3.0 (21 July 2016)
+
+### Features
+
+* use proxy from git config  #1358 (@technoweenie)
```

Comment:
Add `jonmagic` and `lizzhale` since they worked on the original PR. I usually include them in the order of the PR, so `@jonmagic, @lizzhale, @technoweenie` for this item.